zerver/lib/actions.py

```diff
@@ -5532,17 +5532,24 @@ def do_update_message_flags(
         if flag != "starred":
             raise JsonableError(_("Invalid message(s)"))
         # Validate that the user could have read the relevant message
```

Comment:
This comment belongs inside the `transaction.atomic`.

Reply:
Done